website/blog/2022-06-21-version-069.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Announcing React Native 0.69
+authors: [fortmarek, cortinico]
+tags: [announcement, release]
+---
+
+# Announcing 0.69
+
+Hey all, 👋
```

Comment:
Our blog post voice typically avoids these kinds of intros and jumps straight into the content